pkg/portal/graphql/sms_mutation.go

```diff
@@ -98,6 +98,13 @@ var _ = registerMutationField(
 				return nil, err
 			}
 
+			if providerConfig.Twilio != nil {
+				err = gqlCtx.SMSService.SendByTwilio(ctx, app, to, *providerConfig.Twilio)
```

Comment:
The SMS service should expose a single method that takes model.SMSProviderConfigurationInput instead, so we do not need to have a long if-then-else block here.